common/views/components/WorkMedia/WorkMedia2.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import {grid, font, spacing} from '../../../utils/classnames';
+import Icon from '../Icon/Icon';
+import ImageViewer2 from '../ImageViewer/ImageViewer2';
+import ScrollToInfo from '../ScrollToInfo/ScrollToInfo';
+import {iiifImageTemplate} from '../../../utils/convert-image-uri';
+
+type Props = {|
+  id: string,
+  title: string,
+  iiifUrl: string,
+  width?: number,
+  queryString?: string,
+|}
+
+const href = (queryString, id) => {
+  return `/works${queryString}#${id}`;
+};
+
+const tracking = (queryString, id, trackTitle) => {
+  return `{
+    "category": "component",
+    "action": "return-to-results:click",
+    "label": "id:${id}, query:${queryString}, title:${trackTitle}"
+  }`;
+};
+
+const WorkMedia = ({
+  id,
+  title,
+  iiifUrl,
+  width = 800,
+  queryString
+}: Props) => {
+  const trackTitle = title.substring(0, 50);
+  const imageContentUrl = iiifImageTemplate(iiifUrl)({ size: `${width},` });
+  return (
+    <div>
+      {queryString &&
+      <div className="row is-hidden-s is-hidden-m">
```

Comment:
Should this be part of the component?
Feels like it should be part of the template?

Reply:
As above